recipes/mmannot/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "mmannot" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/mzytnicki/mmannot/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4c7f839dd0d040072ca872511b12acbd873a7feeffeea3fe50ba76b968ccd71a
+
+requirements:
+  build:
+    - make
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
+    - make
+
```

Comment:
zlib is already in `host`, and you probably don't need `make` as a runtime dependency.

Reply:
@daler I think I made the requested changes.  Is it better now?

Reply:
@daler I had to add zlib in `host` again.  I had the following error otherwise:

    mmannot.cpp:37:10: fatal error: zlib.h: No such file or directory